src/services/upload/uploadService.ts

```diff
@@ -0,0 +1,259 @@
+import { fileAttribute, FILE_TYPE } from '../fileService';
+import { Collection } from '../collectionService';
+import { logError } from 'utils/sentry';
+import UploadHttpClient from './uploadHttpClient';
+import {
+    extractMetadata,
+    getMetadataMapKey,
+    ParsedMetaDataJSON,
+} from './metadataService';
+import { generateThumbnail } from './thumbnailService';
+import {
+    getFileType,
+    getFileOriginalName,
+    getFileData,
+} from './readFileService';
+import { encryptFiledata } from './encryptionService';
+import { ENCRYPTION_CHUNK_SIZE } from 'types';
+import { uploadStreamUsingMultipart } from './multiPartUploadService';
+import UIService from './uiService';
+import { parseError } from 'utils/common/errorUtil';
+import { MetadataMap } from './uploadManager';
+
+export const MIN_STREAM_FILE_SIZE = 20 * 1024 * 1024;
```

Comment:
MULTIPART_PART_SIZE  
This is the chunk size of a file which will be uploaded in a single part.

Reply:
FILE_READER_CHUNK_SIZE = ENCRYPTION_CHUNK_SIZE